src/Login/Login.jsx

```diff
@@ -66,40 +66,100 @@ class Login extends React.Component {
       : 'gene';
     const customImageStyle = { backgroundImage: `url(/src/img/icons/${customImage}.svg)` };
 
-    const loginOptions = {}; // one for each login provider
-    const filterOptions = {};
-    this.props.providers.forEach((provider, i) => {
+    let loginComponent = (
+      <React.Fragment key='login-component'>
+        <div className='login-page__entries'>
+          <div className='login-page__entry-login'>
+            <Button
+              onClick={() => {}}
+              label='Loading login options'
+              buttonType='primary'
+              isPending
+              enabled={false}
+              tooltipEnabled
+              tooltipText='Getting available login options... If this button has been loading for a while, please try to refresh this page or contact support'
```

Comment:
i'm not a big fan of the tooltip transparency
<img width="653" alt="Screen Shot 2020-06-19 at 2 57 20 PM" src="https://user-images.githubusercontent.com/4224001/85175732-b9e89c80-b23d-11ea-86ee-da98ee7322c4.png">

Reply:
idk i kinda like it =D

Reply:
I think the semi transparent tooltip looks cool but a bit hard to read if the background underneath also has some text on it. So I would vote for a opaque tooltip for readability.

In that case, can I ask any of you for 1 more PR review? @paulineribeyre or @ZakirG https://github.com/uc-cdis/gen3-ui-component/pull/72

Reply:
other tooltips i can find (explorer page filters, dictionary viewer, navbar buttons) are not transparent